### DISCUSSION TITLE
Override username / email address

### DISCUSSION THREAD

**Discussion by M1scer** [authelia/authelia#7739] [Q&A] [answered]
I use Immich and have created an account there (Account1), but I would like to use it with several logins.

Let's say I have created User1 and User2 in Authelia. Regardless of whether I log in with User1 or User2, I would like to be logged in to immich as Account1.

Can I overwrite the account information that is sent to Immich to match the information of Account1?

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for not replying sooner. This feature is not possible to implement technically at our end, as the `sub` claim is the only valid claim for account anchorage and modification of this claim can cause a lot of issues. The correct way to implement this is for Immich to allow users to link multiple accounts to the same account as this is an area of the spec that is generally encouraged.
